packages/controllers/profile_edit.coffee

```diff
@@ -0,0 +1,29 @@
+if Meteor.isClient
+  Template.profileEdit.onCreated ->
+    instance = this
+    subscription = instance.subscribe('userProfile')
+
+  Template.profileEdit.helpers
+    userProfile: ->
+      UserProfiles.findOne()
+
+  Template.profileEdit.events
+    'submit form': (event) ->
+      event.preventDefault()
+      form = event.target
+      fields = {
+        fullName: form.fullName?.value
+        jobTitle: form.jobTitle?.value
+        bio: form.bio?.value
+        emailHidden: form.emailHidden?.value
+      }
+      Meteor.call('updateProfile', fields, -> {})
```

Comment:
I was planning on test-driving the error message while I'm also working on the success message